### PR TITLE
New version: Harmonoid.Harmonoid version 0.3.5.0

### PR DESCRIPTION
**Pull request by alexmercerind**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/93129)

### CODE REVIEW

File: manifests/h/Harmonoid/Harmonoid/0.3.5.0/Harmonoid.Harmonoid.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+- silentWithProgress
+UpgradeBehavior: install
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
+ReleaseDate: 2022-11-16
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/alexmercerind2/harmonoid-releases/releases/download/v0.3.5/harmonoid-windows-setup.exe
+  InstallerSha256: B9B8D153F581460DFA8D0AFB4DEF2EEC6E60ECE6CC0D499FC5713E37D0975D57
```
Comment (stephengillie) [inline manifests/h/Harmonoid/Harmonoid/0.3.5.0/Harmonoid.Harmonoid.installer.yaml:20]:
```suggestion
  InstallerSha256: B9B8D153F581460DFA8D0AFB4DEF2EEC6E60ECE6CC0D499FC5713E37D0975D57
    InstallerSwitches:
      Custom: /allusers
```

Comment (stephengillie) [reply manifests/h/Harmonoid/Harmonoid/0.3.5.0/Harmonoid.Harmonoid.installer.yaml:20]: The installer prompts for installation location - in Program Files or in the user's AppData Local folder. This switch installs it into the recommended location.
image

File: manifests/h/Harmonoid/Harmonoid/0.3.5.0/Harmonoid.Harmonoid.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    InstallerSwitches:
+      Custom: /allusers
```
Comment (stephengillie) [inline manifests/h/Harmonoid/Harmonoid/0.3.5.0/Harmonoid.Harmonoid.installer.yaml:22]:
```suggestion
  InstallerSwitches:
    Custom: /allusers
```

Comment (stephengillie) [reply manifests/h/Harmonoid/Harmonoid/0.3.5.0/Harmonoid.Harmonoid.installer.yaml:22]: Apologies - you were too quick and approved my suggestion before I could correct my spacing error.

Comment (alexmercerind) [reply manifests/h/Harmonoid/Harmonoid/0.3.5.0/Harmonoid.Harmonoid.installer.yaml:22]: Thanks a lot for the review process. Getting it right quickly.

**Comment by alexmercerind** [comment]
Someone raised a pull-request for giving ability to make [user-specific installation](https://github.com/harmonoid/harmonoid/issues/372).
I didn't think this would cause issues with WinGet, so left it as it is since I was tired.

Thanks for the correct solution!

**Review by stephengillie — APPROVED** [review]

**Pull request merged**